python/pyarrow/array.pxi

```diff
@@ -116,10 +117,16 @@ cdef class Array:
         type : pyarrow.DataType
             Explicit type to attempt to coerce to
 
-        timestamps_to_ms : bool, optional
-            Convert datetime columns to ms resolution. This is needed for
-            compatibility with other functionality like Parquet I/O which
-            only supports milliseconds.
+        coerce_timestamps : {'ms', 'us', False}, default False
+            Convert datetime columns to the indicated resolution. If False
+            preserve the existing dtype.
+            This is needed for compability with other functionality like
+            Parquet I/O which only supports milliseconds / microseconds.
```

Comment:
I don't think this is the right fix. The coercion happens in `parquet.write_table`